framework/db/models.py

```diff
@@ -25,7 +25,7 @@ class Session(Base):
     __tablename__ = "sessions"
 
     id = Column(Integer, primary_key=True, autoincrement=True)
-    name = Column(String, unique=True)
+    name = Column(String(50), unique=True)
```

Comment:
In this case, if I bypass the ajax request and still have a session name longer than 50 char, then this will result in an uncaught exception and sqlalchemy will probably crash. Instead, truncate the length here,  https://github.com/owtf/owtf/blob/develop/framework/db/session_manager.py#L60:
'
`session_obj = models.Session(name=session_name[:50])`